apps/billing/src/styles.css

```diff
@@ -193,7 +193,7 @@ sup {
 }
 
 a {
-  color: #6e00ff;
+  color: #2A85D1;
```

Comment:
e2ccff -> cccdff

5800cc -> 0074d3

6e00ff -> 2A85D1

0400ff -> 0051ff

for every rgba -> (0, 116)

5e00d9 -> 0074d9

5300bf -> 0074d9

6300e6 -> 006be6